src/main/java/com/mineboundteam/minebound/item/armor/ArmorTier.java

```diff
@@ -6,10 +6,10 @@
 import net.minecraft.network.chat.TranslatableComponent;
 
 public enum ArmorTier {
-    EFFIGY(0),
-    SUIT(1),
-    SYNERGY(2),
-    SINGULARITY(3);
+    EFFIGY(1), // Tier 1
```

Comment:
Don't do this, if you want the text to increment do it in the toString; in java numbers start at 0

Reply:
Done.

Reply:
Done.